index.html

```diff
@@ -57,6 +59,169 @@ <h1 id="home-section-title">I am Hiro Glad to See You!</h1>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="author-skills">Back End Dev</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="card-date">2015</p>
+            </div>
+            <p class="card-inner-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="skills">
+              <button class="card-skills-button" type="submit">Html</button>
+              <button class="card-skills-button" type="submit">Css</button>
+              <button class="card-skills-button" type="submit">Javascript</button>
+            </div>
+            <button class="see-project-button" type="submit">See Project</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="about-section">
+        <div class="about-section-details">
+          <h1 id="about-section-title">About Myself</h1>
+          <p id="about-section-info">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <div class="about-section-contact">
+            <p class="contact"><a href="#">LET'S CONNECT</a></p>
+            <ul class="social-icons">
+              <li>
+                <a href="#"><i class="fa fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-linkedin"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-github"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram"></i></a>
+              </li>
+            </ul>
+          </div>
+          <button class="about-button" type="submit">Get my resume</button>
+
+          <div class="learned-skills">
+            <div class="languages-select">
+              <p>Languages</p>
+              <i class="fa fa-caret-down"></i>
+            </div>
+          </div>
+          <div class="language-list">
+            <div class="single-language">
+              <img src="/img/js.png" alt="Javascript" />
+              <p>Javascript</p>
+            </div>
+            <div class="single-language">
+              <img src="/img/html.png" alt="Html" />
+              <p>HTML</p>
+            </div>
+            <div class="single-language">
+              <img src="/img/css.png" alt="Html" />
+              <p>CSS</p>
+            </div>
+          </div>
+          <div class="frameworks-select">
+            <p>Frameworks</p>
+            <i class="fa fa-caret-right"></i>
+          </div>
+          <div class="skills-select">
+            <p>Skills</p>
+            <i class="fa fa-caret-right"></i>
+          </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") is a list and should use the `ul` for grouping, while each of them should have a `li` tag
- [ ] The technology in each category (e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`
- [ ] The technology in each category (e.g., "JavasScript", "HTML" and "CSS") should also be embedded inside the `Languages`